src/Locale/Translator.php

```diff
@@ -28,7 +29,6 @@ public function getCatalogue($locale = null)
         } else {
             $this->assertValidLocale($locale);
         }
-
```

Comment:
?

Reply:
Most likely happened during debugging as the Translator instance wasn't being resolved correctly.